### PR TITLE
Mdwfp 55 api endpoints

### PR DESCRIPTION
**Pull request by ldlharper**

### CODE REVIEW

**Review by joelstobart-moj — COMMENTED** [review]
Just a spelling thing - other than that LGTM

**Review by jbowdidge-moj — COMMENTED** [review]
Just one really small comment from me on custom media types :) Other than LGTM

File: src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/legacy/controller/LegacyController.kt [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.ResponseStatus
+import org.springframework.web.bind.annotation.RestController
+import uk.gov.justice.digital.hmpps.adjustments.api.legacy.model.LegacyAdjustment
+import uk.gov.justice.digital.hmpps.adjustments.api.legacy.model.LegacyAdjustmentCreatedResponse
+import uk.gov.justice.digital.hmpps.adjustments.api.legacy.service.LegacyService
+import java.util.UUID
+
+@RestController
+@RequestMapping("/adjustments", produces = [MediaType.APPLICATION_JSON_VALUE], consumes = ["application/nomis_offence+json"])
```
[the remark this answers is not in the document]
Comment (joelstobart-moj) [reply src/main/kotlin/uk/gov/justice/digital/hmpps/adjustments/api/legacy/controller/LegacyController.kt:20]: Good spot Jamie (that was my mistake)

**Review by joelstobart-moj — DISMISSED** [review]
LGTM

**Review by jbowdidge-moj — APPROVED** [review]

**Pull request merged**